pkg/manager/impl/project_manager.go

```diff
@@ -28,7 +27,7 @@ func (m *ProjectManager) CreateProject(ctx context.Context, request admin.Projec
 	if err := validation.ValidateProjectRegisterRequest(request); err != nil {
 		return nil, err
 	}
-	projectModel := transformers.CreateProjectModel(request.Project.Id, request.Project.Name)
+	projectModel := transformers.CreateProjectModel(request.Project.Id, request.Project.Name, request.Project.Description)
```

Comment:
nit: might just be simpler to pass in a pointer to request.Project instead of decomposing nearly all the fields

Reply:
That does make sense. I think I was blindly extending the existing pattern :-)